aten/src/ATen/native/native_functions.yaml

```diff
@@ -8649,6 +8655,11 @@
     CPU: col2im_backward_cpu
     CUDA: col2im_backward_cuda
 
+- func: column_stack(Tensor[] tensors) -> Tensor
+  use_c10_dispatcher: full
+
```

Comment:
Same dispatch comment her as for row_stack (since this is a composite op).

Reply:
Math dispatch keys are added for `column_stack` and `column_stack.out` as well.